neovim/plugin/script_host.py

```diff
@@ -59,9 +60,6 @@ def setup(self, nvim):
 
     def teardown(self):
         """Restore state modified from the `setup` call."""
-        for plugin in self.installed_plugins:
```

Comment:
Actually, it was there for supporting teardown hooks so plugins had a chance to cleanup or save state when the host is closes, but I guess it was unused so it should be fine to remove.